weasyl/controllers/api.py

```diff
@@ -9,7 +9,7 @@
 from weasyl import define as d, macro as m
 from weasyl import (
     api, character, collection, commishinfo, folder, media, message, profile,
-    index, searchtag, submission, favorite)
+    index, searchtag, submission, journal, favorite)
```

Comment:
Nit: The multiple statements are fine, but they should be ordered (which means fixing index and favorite here).